MapboxDirections/MBDirections.swift

```diff
@@ -103,7 +103,7 @@ open class Directions: NSObject {
         If the request was canceled or there was an error obtaining the routes, this parameter is `nil`. This is not to be confused with the situation in which no results were found, in which case the array is present but empty.
      - parameter error: The error that occurred, or `nil` if the placemarks were obtained successfully.
```

Comment:
TODO: 

- [ ] Add a description for the new field `_ JSONRoutes: [[String: Any]]?` within the `RouteCompletionHandler`